test/server/controllers/conversations.test.ts

```diff
@@ -0,0 +1,100 @@
+                    .send(createConversationClone).expect(200)
+                expect(res.body.conversations.length).to.eq(1)
+                expect(res.body.conversations[0].from.id).to.eq(createConversationClone.fromId)
+                expect(res.body.conversations[0].to.id).to.eq(createConversationClone.toId)
+            })
+            it('Should get an empty conversation list', async () => {
+                const token = (await logInUser(user)).accessToken
+                const res = await request.get('/api/conversations/').set('Authorization', `bearer ${token}`)
+                    .send(createConversationClone).expect(200)
+                expect(res.body.conversations.length).to.eq(0)
+            })
+        })
+    })
+
+    describe('#GET /api/conversations/:conversationId', async () => {
+        context('Conversation already created', async () => {
+            let createConversationClone, conversationId, user, user2
+            beforeEach(async () => {
+                user = await factory.create('user', createUser)
+                user2 = await factory.create('user', createUser2)
+                createConversationClone = _.cloneDeep(createConversation)
+                createConversationClone.fromId = user.id
+                createConversationClone.toId = user2.id
+                const conversation = await factory.create('conversation', createConversationClone)
+                conversationId = conversation.id
+            })
+            it('Should get the conversation', async () => {
+                const token = (await logInUser(user)).accessToken
+                const res = await request.get(`/api/conversations/${conversationId}`).expect(200)
+                expect(res.body.conversation.id).to.eq(conversationId)
+                expect(res.body.conversation.from.id).to.eq(createConversationClone.fromId)
+                expect(res.body.conversation.to.id).to.eq(createConversationClone.toId)
+            })
+        })
+        context('No conversation previously created', async () => {
+            let createConversationClone, user, user2
+            beforeEach(async () => {
+                user = await factory.create('user', createUser)
+                user2 = await factory.create('user', createUser2)
+                createConversationClone = _.cloneDeep(createConversation)
+                createConversationClone.fromId = user.id
+                createConversationClone.toId = user2.id
+            })
+            it.only('Should not get the conversation', async () => {
+                const conversationId: string = uuid()
+                const token = (await logInUser(user)).accessToken
+                const res = await request.get(`/api/conversations/${conversationId}`).expect(404)
+                expect(res.body.message).to.eq('Conversation not found')
+            })
+        })
+    })
+
+    describe('#POST /api/conversations/', async () => {
```

Comment:
Deberíamos controlar que no puede haber más de una conversación entre 2 usuarios. Creo que esto no se está haciendo en el BE hoy en día pero debería de ser un caso de prueba que probamos y reaccionamos ante él